reactioncommerce/example-storefront
# Feature/stripe sca

Impact: **major**
Type: **feature**

## Issue
Stripe SCA needs to be configured by default, specially for European users.

## Solution

Stripe SCA plugin is now available in the storefront

## Breaking changes

none, old Stripe plugin will continue to work but it will be deprecated in 10/31/2021.

## Testing

1. Set up Stripe private key in API Environment keys (.env)
2. Set up stripe public key in  storefront environment keys
3. set a shipment method.
4. Activate Stripe as payment method. version 1.0.0 of the plugin will say stripe, subsequent version will say Stripe (SCA) to avoid confusion with old plugin.
5.  go to step 3 of checkout.
6. watch new Stripe form appear (or both options in a radio selection) and the selected on in a form.
